### PR TITLE
add ingredients

### PR DESCRIPTION
**Pull request by Marie-Sail**

### CODE REVIEW

**Review by Thomas-Nigon — APPROVED** [review]

File: backend/src/controllers/recipeControllers.js [hunk 1]
```diff
@@ -1,128 +1,129 @@
...
       userId,
       stepsArr,
-      ingredientArr,
       diet,
       type,
       season
     );
 
-    res.json(insertId);
+    // eslint-disable-next-line prefer-const
```
Comment (fantinerudent) [inline backend/src/controllers/recipeControllers.js:21]: why you did not put a "const" if it ask you to?

File: backend/src/controllers/recipeControllers.js [hunk 2]
```diff
@@ -1,128 +1,129 @@
...
+    next(err);
+  }
+};
+
+const uploadPicture = async (req, res, next) => {
+  try {
+    if (!req.file) {
+      return res.status(400).send("No file uploaded.");
+    }
+    // Le nom du fichier stocké est accessible via req.file.filename
```
Comment (fantinerudent) [inline backend/src/controllers/recipeControllers.js:44]: english plz

File: backend/src/middleware/upload.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
+// eslint-disable-next-line import/no-unresolved, import/no-extraneous-dependencies
+const multer = require("multer");
+const path = require("path");
+
+const storage = multer.diskStorage({
+  destination: (req, file, cb) => {
+    // Chemin de destination des fichiers uploadés
```
Comment (fantinerudent) [inline backend/src/middleware/upload.js:7]: english plz

File: backend/src/middleware/upload.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+    cb(null, "uploads");
+  },
+  filename: (req, file, cb) => {
+    // Construire le nom du fichier avec son nom d'origine et l'extension d'origine, autrement le fichier ne possède pas d'extension
```
Comment (fantinerudent) [inline backend/src/middleware/upload.js:11]: same

**Review by fantinerudent — APPROVED** [review]

**Pull request merged**